release-content/0.14/release-notes/13121_Implement_filmic_color_grading.md

```diff
@@ -1,4 +1,16 @@
 <!-- Implement filmic color grading. -->
 <!-- https://github.com/bevyengine/bevy/pull/13121 -->
 
-<!-- TODO -->
+Artists want to get exactly the right look for their game, and color plays a huge role.
+
+To support this, Bevy's [existing tonemapping tools](https://bevyengine.org/news/bevy-0-10/#more-tonemapping-choices) have been extended to include a complete set of filmic color grading tools. In addition to a [base tonemap](https://dev-docs.bevyengine.org/bevy/core_pipeline/tonemapping/enum.Tonemapping.html), you can now configure:
+
+- White point adjustment. This is inspired by Unity's implementation of the feature, but simplified and optimized. Temperature and tint control the adjustments to the x and y chromaticity values of CIE 1931. Following Unity, the adjustments are made relative to the D65 standard illuminant in the LMS color space.
+- Hue rotation: converts the RGB value to HSV, alters the hue, and converts back.
+- Color correction: allows the gamma, gain, and lift values to be adjusted according to the standard ASC CDL combined function. This can be done seperately for shadows, midtones and highlights To avoid abrupt color changes, a small crossfade is used between the different sections of the image.
```

Comment:
```suggestion
- Color correction: allows the gamma, gain, and lift values to be adjusted according to the standard ASC CDL combined function. This can be done separately for shadows, midtones and highlights To avoid abrupt color changes, a small crossfade is used between the different sections of the image.
```